packages/vuepress-theme-vt/layouts/Layout.vue

```diff
@@ -154,7 +182,20 @@ export default {
           this.toggleSidebar(false)
         }
       }
-    }
-  }
-}
+    },
+
+    getPageLayout() {
+      if (this.$page.path) {
```

Comment:
What does this `this.$page.path` mean ?

Reply:
Copied from VuePress but I forgot the motivation (so long ago), maybe I can remove it.